core/src/test/java/dk/alexandra/fresco/framework/util/TestDetermSecureRandom.java

```diff
@@ -1,11 +1,11 @@
 package dk.alexandra.fresco.framework.util;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertTrue;
 
+import org.junit.Test;
 import dk.alexandra.fresco.framework.MPCException;
+import static org.junit.Assert.assertArrayEquals;
```

Comment:
Static imports should be at the top

Reply:
Fixed.

Reply:
Sorry, but static and non static should be in separate blocks and each block should be ordered alphabetically. https://google.github.io/styleguide/javaguide.html#s3.3-import-statements . It's really easier to do this organization automatically. Let me know if you need help setting up the IDE.